pandas/tests/frame/methods/test_replace.py

```diff
@@ -21,6 +21,50 @@ def mix_abc() -> Dict[str, List[Union[float, str]]]:
     return {"a": list(range(4)), "b": list("ab.."), "c": ["a", "b", np.nan, "d"]}
 
 
+@pytest.fixture(
+    params=[
+        "any_real_dtype",
+        "string_dtype",
+        "datetime64_dtype",
+        "timedelta64_dtype",
+        "bool",
+        "complex_dtype",
+        "bytes_dtype",
+        "tz_aware_fixture",
```

Comment:
i think i see what this is doing, but at first glance it looks like a weird usage.  can you add a docstring for when i inevitably forget?